src/main/proto/com/kodypay/grpc/pay/v1/pay.proto

```diff
@@ -31,6 +31,20 @@ message PayRequest {
   string amount = 2; // amount in BigDecimal/2.dp (0.00)
   string terminal_id = 3; // send the payment to this terminal serial number
   optional bool show_tips = 4; // Show tips on the terminal
+  PaymentFlow payment_flow = 5; // Choose specific payment flow on the terminal
```

Comment:
is it possible to make this optional so we keep backwards compatibility?
The default would be Card Present payment type